README.md

```diff
@@ -1,4 +1,9 @@
 # react-simple-infinite-loading
+[![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
```

Comment:
There are 5 badges could you keep only the first one ?

Reply:
ok, i did it. Can you review code again?